.eslintrc

```diff
@@ -45,6 +45,7 @@
 		"@typescript-eslint/prefer-nullish-coalescing": ["error"],
 		// Enabling TS rule and disabling Base rule as it can report incorrect errors.
 		"no-shadow": "off",
-		"@typescript-eslint/no-shadow": ["error"]
+		"@typescript-eslint/no-shadow": ["error"],
+		"@wordpress/no-unsafe-wp-apis": "off"
```

Comment:
Yes, only for the experimental components. As far as I'm aware, that's the only thing that this specific ESLint rule checks for.

Reply:
I'd maybe put a comment there. Something like `// Allow usage of Gutenberg experimental components`. Not required, but add it if you think it could be useful.